app/controllers/api/v0/base_controller.rb

```diff
@@ -2,44 +2,43 @@ module Api
   module V0
     class BaseController < ApplicationController
       def list_order_item
-        item = OrderItem.where('id = ? and order_id = ?',
-                               params['order_item_id'], params['order_id']).first
-        render json: item.to_hash
+        render json: Order.find(params.require(:order_id)).
+          order_items.find(params.require(:order_item_id)).to_hash
```

Comment:
Why `to_hash`?  Isn't this handled by the `render json` calling `as_json`?

Reply:
@bdunne That was an earlier implementation we are planning on using a generic one that the topology service has which looks at the JSON schema from the swagger definition and does the proper conversion. Didn't want to muddy the waters by including it here. We we will be doing the to_hash cleanup in a separate PR

Reply:
Tracking this work in https://projects.engineering.redhat.com/browse/SSP-95